src/test/scala/io/iohk/ethereum/ledger/BlockExecutionSpec.scala

```diff
@@ -49,7 +49,8 @@ class BlockExecutionSpec extends AnyWordSpec with Matchers with ScalaCheckProper
         val blockExecution =
           new BlockExecution(blockchain, blockchainConfig, newConsensus.blockPreparator, blockValidation)
 
-        val (blocks, error) = blockExecution.executeAndValidateBlocks(List(block1, block2), defaultBlockHeader.difficulty)
+        val (blocks, error) =
+          blockExecution.executeAndValidateBlocks(List(block1, block2), defaultBlockHeader.difficulty)
```

Comment:
Are you maybe using different scalafmt rules? (I'm just wondering the changes on this file)

Reply:
I don't know, I just ran it `sbt test:scalafmt` and it changed these files.

Reply:
Seems to make sense though, since the [maxColumn](https://github.com/input-output-hk/mantis/blob/develop/.scalafmt.conf#L3) setting is 120 and this line was over that.